examples/app/Embeddings.ipynb

```diff
@@ -7,8 +7,36 @@
    "source": [
     "## Embedding explorer\n",
     "\n",
-    "- Embeds a string column using OpenAI Embeddings\n",
-    "- Follows the clustering / projection approach from here: https://umap-learn.readthedocs.io/en/latest/clustering.html"
+    "In this tutorial, we'll create a notebook-based Embeddings explorer. We will use functions, or `Ops`, defined in the weave ecosystem so that they're available to compatible types in the Weave UI.\n",
+    "\n",
+    "You'll learn how to:\n",
+    "- Save and version embeddings\n",
```

Comment:
nit: not sure we're actually versioning embeddings yet—if we
1. set up a side-by-side comparison with different hyperparameter settings
2. describe how to give those different names
then we'll have this!